src/components/AddressInput.js

```diff
@@ -143,7 +144,7 @@ class AddressInput extends React.Component {
           />
         </FormLabelGroup>
         <Row className="no-gutters">
-          <Col sm={6} xs={12} className="pr-sm-3">
+          <Col sm={compact ? undefined : 6} xs={12} className="pr-sm-3">
```

Comment:
I think we should get rid of the `pr` class when in compact mode similar to the col size logic:

```
<Col sm={compact ? undefined : 6} xs={12} className={compact ? undefined: 'pr-sm-3'}>
```

Otherwise can lead to some funky padding when the viewport size is >= `sm`

![Screen Shot 2021-09-28 at 5 09 26 PM](https://user-images.githubusercontent.com/759759/135172861-328498c0-b501-44a5-b535-86e8fa6cf890.png)